src/main/java/com/openlattice/organization/OrganizationsApi.java

```diff
@@ -47,6 +49,7 @@
     String ID_PATH           = "/{" + ID + "}";
     String INTEGRATION       = "/integration";
     String MEMBERS           = "/members";
+    String METADATA_ENTITY_SETS = "/metadata-entity-sets";
```

Comment:
```
String METADATA_ENTITY_SET_IDS = "/metadata-entity-set-ids";
```

Reply:
Or, alternatively, just `/metadata` if we anticipate the `setMetadataEntitySetIds` endpoint to possibly support a more complex metadata object in the future.